.github/workflows/update-lib.yaml

```diff
@@ -0,0 +1,27 @@
+name: Update ip-router-interface
```

Comment:
Actually I just realised, why not leverage the common workflow like in [Kubernetes charm libraries](https://github.com/canonical/kubernetes-charm-libraries/blob/main/.github/workflows/publish-libs.yaml):
- canonical/sdcore-github-workflows/.github/workflows/publish-lib.yaml@main

Reply:
Didn't see this example in the repo i looked at. Updated.